### DISCUSSION TITLE
Patch for E31 version

### DISCUSSION THREAD

**Discussion by excalibur25** [Arkady23/V380-Web#21] [General]
Hi,

is it possible to modify the patch for a E31 version?
I have a camera running these firmware.

Softwareversion: AppV380E31_V1.1.1.6_20210719
Firmwareversion: HwV380E31_WF3_IPC60_ETH_20201218

Thank you and Best Regards
Christian

**Comment by Arkady23** [comment]
Do you have any firmware from your camera. Or can you order the firmware on support, as [hrpoint did](https://github.com/Arkady23/V380-Web/discussions/20)?

**Reply by excalibur25** [reply]
Thank you for the fast answer. Unfortunately I have no firmware yet. I will try to get one from the support asap.

**Reply by excalibur25** [reply]
I contacted the v380technical support but are still waiting for feedback.

**Reply by excalibur25** [reply]
[AK_3918E-V330__V1.1.3.3_2021-10-26.zip](https://github.com/Arkady23/V380-Web/files/7833909/AK_3918E-V330__V1.1.3.3_2021-10-26.zip)
[SPECIAL_V380_AK_3918E-V330_20MB-DMA_V9.9.9.9_2021-09-24 (1).zip](https://github.com/Arkady23/V380-Web/files/7833911/SPECIAL_V380_AK_3918E-V330_20MB-DMA_V9.9.9.9_2021-09-24.1.zip)

**Reply by excalibur25** [reply]
They sent me two files for upgrading, the special should be for rtsp fixing. But this port is still not open after uprgrade.

**Reply by excalibur25** [reply]
ok, after applying the ceshi.ini with rtsp=1 and reset the RTSP stream is working. Would be nice to have the webinterface from you too :)

**Reply by Arkady23** [reply]
https://github.com/Arkady23/V380-Web/releases/download/20210530/httpd_V380E4_C3_9090909_v20210620.zip

**Comment by daoudeddy** [comment]
Hello i got a firmware for V380E4_C3 can you make your patch for it? Thank you

[正式升级包_V380_安凯_3918E-V330__V1.1.4.0_2022-01-05.zip](https://github.com/Arkady23/V380-Web/files/8231138/_V380_._3918E-V330__V1.1.4.0_2022-01-05.zip)

**Reply by Arkady23** [reply]
So there is such a patch [in the table](https://github.com/Arkady23/V380-Web#%D1%82%D0%B0%D0%B1%D0%BB%D0%B8%D1%86%D0%B0-%D0%BE%D0%BF%D1%80%D0%B5%D0%B4%D0%B5%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F-%D0%BC%D0%BE%D0%B4%D0%B5%D0%BB%D0%B8-%D0%BA%D0%B0%D0%BC%D0%B5%D1%80%D1%8B-v380) Last line.

**Reply by daoudeddy** [reply]
It didnt work i dont know why
I tried the firmware above got the voice firmware confirmation, but when I try your firmware it boots normally without entering the firmware update on boot

**Comment by Arkady23** [comment]
Added a description for installing a web server manually for new cameras.
